src/main/webapp/resources/js/pages/sequencing-runs2/components/SequencingRunDetailsPage.jsx

```diff
@@ -0,0 +1,135 @@
+import {
+  useGetSequencingRunDetailsQuery,
+  useGetSequencingRunFilesQuery
+} from "../../../apis/sequencing-runs/sequencing-runs";
+import { Badge, Button, Col, Row, Table, Typography } from "antd";
+import { formatDate } from "../../../utilities/date-utilities";
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { IconDownloadFile } from "../../../../js/components/icons/Icons";
+import { LinkButton } from "../../../components/Buttons/LinkButton";
+import { BasicList } from "../../../components/lists";
+import { NarrowPageWrapper } from "../../../components/page/NarrowPageWrapper";
+
+/**
+ * React component to display the sequencing run details page.
+ * @returns {*}
+ * @constructor
+ */
+export default function SequencingRunDetailsPage() {
+  const isTechnician = window.TL._USER.systemRole === "ROLE_TECHNICIAN";
+  const {runId} = useParams();
+  const {
+    data: run = {},
+    isLoading: isRunLoading
+  } = useGetSequencingRunDetailsQuery(runId);
+  const {
+    data: files = [],
+    isLoading: isFilesLoading
+  } = useGetSequencingRunFilesQuery(runId);
+  const detailsList = isRunLoading
+    ? []
+    : [
+      {
+        title: "Sequencer Type",
+        desc: run.sequencerType
+      },
+      {
+        title: "Upload Status",
+        desc: <UploadStatusBadge status={run.uploadStatus}/>
+      },
+      {
+        title: "Upload User",
+        desc: isTechnician ? run.userName : (<LinkButton
+            text={run.userName}
+            href={setBaseUrl(`/users/${run.userId}`)}/>
+        )
+      },
+      {
+        title: "Created",
+        desc: formatDate({date: run.createdDate})
+      },
+      {
+        title: "Description",
+        desc: run.description
+      },
+    ];
+  const optionalPropertiesList = [];
+  run.optionalProperties && Object.entries(run.optionalProperties).map(([key, value]) => (
+    optionalPropertiesList.push({title: key, desc: value})))
+
+  const columns = [
+    {
+      title: 'Filename',
+      dataIndex: 'fileName',
+      key: 'fileName',
+      render(text, item) {
+        return <LinkButton
+          text={text}
+          href={setBaseUrl(`/sequencingRuns/${runId}/sequenceFiles/${item.sequencingObjectId}/file/${item.id}/summary`)}
+        />
+      },
+    },
+    {
+      title: 'Size',
+      dataIndex: 'fileSize',
+      key: 'fileSize',
+      align: 'right'
+    },
+    {
+      title: 'Download',
+      dataIndex: 'download',
+      key: 'download',
+      render(text, item) {
+        return <Button shape="circle"
+                       onClick={() => window.open(setBaseUrl(`/sequenceFiles/download/${item.sequencingObjectId}/file/${item.id}`), "_blank")}
+                       icon={<IconDownloadFile/>}/>;
+      },
+      align: 'center'
+    }
+  ]
+
+  return (
+    <NarrowPageWrapper title={i18n("SequenceRunDetailsPage.title", runId)}>
+      <Row justify="center" gutter={[0, 16]}>
+        <Col span={18}>
```

Comment:
Why did you bring everything smaller than the narrow width?  That si the purpose of the narrow width.